coinflip/main.c

```diff
@@ -0,0 +1,394 @@
+#include <libopencm3/cm3/common.h>
+#include <libopencm3/cm3/vector.h>
+#include <libopencm3/cm3/scb.h>
+#include <libopencm3/efm32/cmu.h>
+#include <libopencm3/efm32/gpio.h>
+#include <libopencm3/efm32/common/prs_common.h>
+#include <libopencm3/efm32/common/acmp_common.h>
+#include <libopencm3/efm32/timer.h>
+#include <libopencm3/efm32/wdog.h>
+
+#include <stdbool.h>
+#include <stdio.h>
+#include <string.h>
+
+#include "captouch.h"
+#include "capsenseconfig.h"
+
+// Make this program compatible with Toboot-V2.0
+#include <toboot.h>
+TOBOOT_CONFIGURATION(TOBOOT_CONFIG_FLAG_AUTORUN);
```

Comment:
I think it's better to leave it 0, and add a comment with this flag, as was done for [ledpulse](https://github.com/im-tomu/tomu-quickstart/blob/master/ledpulse/ledpulse.c).
Saves people looking for tweezers 😅

Reply:
That makes sense to me.